src/BasketBundle/Controller/Api/BasketController.php

```diff
@@ -355,10 +355,9 @@ public function deleteBasketBasketelementsAction($basketId, $elementId)
         }
 
         $view = \FOS\RestBundle\View\View::create($basket);
-        $serializationContext = SerializationContext::create();
-        $serializationContext->setGroups(array('sonata_api_read'));
-        $serializationContext->enableMaxDepthChecks();
-        $view->setSerializationContext($serializationContext);
+        $context = new Context();
```

Comment:
At the current moment in 2.x there are `"friendsofsymfony/rest-bundle": "^1.1"` and `"sonata-project/media-bundle": "^3.0"`, but SonataMediaBundle 3.0.0 requires `"friendsofsymfony/rest-bundle": "^1.7.4 || ^2.0",` so there is no support for older rest-bundle versions.
Thats why I increased to `"friendsofsymfony/rest-bundle": "^1.8"` and removed `SerializationContext`, because it is removed in rest-bundle 2.0